support-internationalisation/src/main/scala/com/gu/i18n/Country.scala

```diff
@@ -5,6 +5,8 @@ package com.gu.i18n
 case class Country(alpha2: String, name: String, statesByCode: Map[String, String] = Map.empty) {
   // Backwards compatibility, including using the 2/3-character codes for Australian states as the name
   val states: Seq[String] = if (alpha2 == "AU") statesByCode.keys.toSeq.sorted else statesByCode.values.toSeq.sorted
+
+  override def toString: String = s"com.gu.i18n.Country.$alpha2"
```

Comment:
these probably shouldn't be modelled as a case class, but the debug from tests was very hard to deal with so I went for the quick fix on toString

after

Left(CreditCardReferenceTransaction(card_EdajV2eXkZPrVV,cus_EdajoRmjUSlef9,************4242,Some(com.gu.i18n.Country.US),2,2022,Some(Visa),StripeGatewayPaymentIntentsDefault,CreditCardReferenceTransaction,None))
 was not equal to 
Left(CreditCardReferenceTransaction(card_EdajV2eXkZPrVV,cus_EdajoRmjUSlef9,************4242,Some(com.gu.i18n.Country.US),2,2022,Some(Visa),StripeGatewayDefault,CreditCardReferenceTransaction,None))
 (PreparePaymentMethodForReuseSpec.scala:40)


before

Left(CreditCardReferenceTransaction(card_EdajV2eXkZPrVV,cus_EdajoRmjUSlef9,************4242,Some(Country(US,United States,Map(MA -> Massachusetts, IN -> Indiana, ID -> Idaho, NM -> New Mexico, OR -> Oregon, IA -> Iowa, IL -> Illinois, TN -> Tennessee, PR -> Puerto Rico, MO -> Missouri, ME -> Maine, AZ -> Arizona, AK -> Alaska, GU -> Guam, VT -> Vermont, WA -> Washington, SD -> South Dakota, KY -> Kentucky, NJ -> New Jersey, TX -> Texas, MI -> Michigan, MD -> Maryland, AA -> Armed Forces America, NV -> Nevada, NE -> Nebraska, MN -> Minnesota, KS -> Kansas, OK -> Oklahoma, CT -> Connecticut, OH -> Ohio, AE -> Armed Forces, AR -> Arkansas, FL -> Florida, WI -> Wisconsin, CO -> Colorado, MT -> Montana, DC -> Washington DC, ND -> North Dakota, PA -> Pennsylvania, GA -> Georgia, NH -> New Hampshire, HI -> Hawaii, WY -> Wyoming, AP -> Armed Forces Pacific, VI -> Virgin Islands, LA -> Louisiana, CA -> California, UT -> Utah, AL -> Alabama, WV -> West Virginia, VA -> Virginia, NC -> North Carolina, NY -> New York, SC -> South Carolina, RI -> Rhode Island, MS -> Mississippi, DE -> Delaware))),2,2022,Some(Visa),StripeGatewayPaymentIntentsDefault,CreditCardReferenceTransaction,None))
 was not equal to 
Left(CreditCardReferenceTransaction(card_EdajV2eXkZPrVV,cus_EdajoRmjUSlef9,4242,Some(Country(US,United States,Map(MA -> Massachusetts, IN -> Indiana, ID -> Idaho, NM -> New Mexico, OR -> Oregon, IA -> Iowa, IL -> Illinois, TN -> Tennessee, PR -> Puerto Rico, MO -> Missouri, ME -> Maine, AZ -> Arizona, AK -> Alaska, GU -> Guam, VT -> Vermont, WA -> Washington, SD -> South Dakota, KY -> Kentucky, NJ -> New Jersey, TX -> Texas, MI -> Michigan, MD -> Maryland, AA -> Armed Forces America, NV -> Nevada, NE -> Nebraska, MN -> Minnesota, KS -> Kansas, OK -> Oklahoma, CT -> Connecticut, OH -> Ohio, AE -> Armed Forces, AR -> Arkansas, FL -> Florida, WI -> Wisconsin, CO -> Colorado, MT -> Montana, DC -> Washington DC, ND -> North Dakota, PA -> Pennsylvania, GA -> Georgia, NH -> New Hampshire, HI -> Hawaii, WY -> Wyoming, AP -> Armed Forces Pacific, VI -> Virgin Islands, LA -> Louisiana, CA -> California, UT -> Utah, AL -> Alabama, WV -> West Virginia, VA -> Virginia, NC -> North Carolina, NY -> New York, SC -> South Carolina, RI -> Rhode Island, MS -> Mississippi, DE -> Delaware))),2,2022,Some(Visa),StripeGatewayDefault,CreditCardReferenceTransaction,None))
 (PreparePaymentMethodForReuseSpec.scala:40)